MANIFEST.in

```diff
@@ -18,3 +18,4 @@ recursive-exclude picard *.pyc
 
 recursive-exclude doc *
 recursive-exclude matlab_octave *
```

Comment:
this line is unecessary now that you moved things

Reply:
the picard algorithm is still in this directory, so we need it.
only the examples moved